interfaces/cython/cantera/reactor.pyx

```diff
@@ -911,21 +911,57 @@ cdef class WallBase:
         def __set__(self, double value):
             self.wall.setArea(value)
 
+    @property
```

Comment:
Can the time value here just take a default value that's checked at runtime to avoid defining this method then immediately deleting it? Especially because the warning indicates that people need to change but then change back what their code is doing.

Reply:
This is a valid concern. However, I'd rather leave it as is for the following two reasons:

- 1 `ExtensibleReactor` recently introduced these as a property, see: https://github.com/Cantera/cantera/blob/d717e8bc6f20afd5dba97b248016e9a9e12f365e/interfaces/cython/cantera/reactor.pyx#L654-L670
- 2. It's not really a commonly used part of the API (there isn't a single example that accesses either of the methods in question).

Reply:
Based on @speths's comments, this is now renamed to `expansion_rate`, where only a single replacement is necessary.

Reply:
Hooray @speth for having the solution 😄